<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThermometryReconstruction</class>
 <widget class="QMainWindow" name="ThermometryReconstruction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThermometryReconstruction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="q_h_layout_viewer">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QVTKOpenGLWidget" name="q_vtk_viewerLeft">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QVTKOpenGLWidget" name="q_vtk_viewerRight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="q_h_layout_fileLoad">
        <item>
         <widget class="QLineEdit" name="q_lineEdit_filename">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="q_pushButton_browse">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="q_h_layout_buttons">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="q_pushButton_defineReference">
          <property name="text">
           <string>Compute Final Nekrose</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="q_pushButton_computeHeatMap">
          <property name="text">
           <string>Simulate Ablation</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
